components/compliance-service/reporting/relaxting/nodes.go

```diff
@@ -200,9 +199,12 @@ func (backend *ES2Backend) GetNode(nodeUuid string) (*reportingapi.Node, error)
 		"status",
 		"controls_sums")
 
+	filters["node_id"] = []string{nodeUuid}
+	query := backend.getFiltersQuery(filters, false)
```

Comment:
I *think* false is the value I want for the second argument, but I'm not really sure. That value becomes the `latestOnly` value used to choose a branch of this conditional: https://github.com/chef/automate/blob/master/components/compliance-service/reporting/relaxting/reports.go#L792-L799

My reasoning is that passing in true would add an additional filter that is not present currently.